lib/commanded/event/handler.ex

```diff
@@ -423,6 +496,32 @@ defmodule Commanded.Event.Handler do
               | {:error, :already_seen_event}
               | {:error, reason :: any()}
 
+  @doc """
+  Handle a batch of domain events and their metadata.
+
+  Return `:ok` on success. All events in the batch will be acknowledged.
+
+  On error, you can either return `{:error, reason}` to indicate something
+  with the whole batch went wrong, or `{:error, reason, event}` to indicate
+  that all events prior to this event were processed successfully but this
+  particular event went wrong.
+
+  In both cases, the error handler will be called. When just a reason is
+  returned, the assumption is that a system issue is preventing the process
+  from continuing and the error handler will be called with a `nil` event
+  argument. When an event and a reason are returned, the assumption is that
+  something is wrong with the event itself and therefore, it is passed in
+  as the first argument for the error handler.
```

Comment:
Looks like this was mismatched docs and implementation. Currently when no specific event is passed, we send the last event in the batch to allow skipping the entire batch. I've updated these docs with the assumption of that behaviour being ok. Otherwise we can update it to send the first event (and the docs accordingly)